services/slam/builtin/builtin.go

```diff
@@ -735,14 +735,22 @@ func (slamSvc *builtIn) getLazyPNGImage(ctx context.Context, cam camera.Camera)
 		return nil, nil, err
 	}
 
-	lazyImg, ok := img.(*rimage.LazyEncodedImage)
-	if !ok {
-		return nil, nil, errors.Errorf("expected lazily encoded image, got %T", lazyImg)
+	if lazyImg, ok := img.(*rimage.LazyEncodedImage); ok {
```

Comment:
Do I understand correctly that for realsense (mono or RGBD), we will still get a lazy PNG here, so there is no difference in the images passed to orbslam?

Reply:
That is correct.